### PR TITLE
Replace alert in MDN/Writing_guidelines/Page_structures/Live_samples

### PR DESCRIPTION
**Pull request by evelinabe**
### Description

Replace alert in a simple introduction to MDN Live examples. Instead of an alert this PR suggests toggling the text instead, since it's not the alert that is in focus, it is showing a simple DOM manipulation

### Motivation

The current example is broken. By replacing the alert with toggling the text most elements are kept as they are, but getting the bonus of a working example of a DOM manipulation.

### Related issues and pull requests

Relates to [#7566](https://github.com/mdn/content/issues/7566)

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md [hunk 1]
```diff
@@ -264,12 +264,16 @@ The CSS code styles the box as well as the text inside it.
 
 #### JavaScript
 
-This code is very simple. All it does is attach an event handler to the "Hello world!" text that makes an alert appear when it is clicked.
+This code is very simple. All it does is attach an event handler to the "Hello world!" text that toggles it when it is clicked.
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md:267]: I'd remove the use of "simple" because this is subjective :)

**Review by bsmth — APPROVED** [review]
Great! One comment open for you to have a look at, but we're good to go then!

File: files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (evelinabe) [reply files/en-us/mdn/writing_guidelines/page_structures/live_samples/index.md]: I've pushed an update

**Comment by bsmth** [comment]
Looks good, merging shortly! Thanks for your contribution!

**Pull request merged**